unfoldingWord/translationCore
# Fix Static Folder Reference

#### This pull request addresses:
Failing build issue

#### How to test this pull request:
Test opening projects and make sure resources copy over


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1927)
